test/integration/smart_answer_flows/benefit_cap_calculator_test.rb

```diff
@@ -11,6 +11,12 @@ class BenefitCapCalculatorTest < ActiveSupport::TestCase
   end
 
   context "Benefit cap calculator" do
+    setup do
+      WebMock.stub_request(:get, "#{Plek.new.find('imminence')}/areas/IG6%202BA.json").
+        to_return(body: File.open(fixture_file('imminence/london.json')))
+      WebMock.stub_request(:get, "#{Plek.new.find('imminence')}/areas/B1%201PW.json").
+        to_return(body: File.open(fixture_file('imminence/national.json')))
```

Comment:
@ikennaokpala has addressed this in a new commit titled "Reuse imminence_has_areas_for_postcode in place of web mock" (174d0813933904f23d2e4bfe7852fbc5da36b58d).